spec/express/testAppSpec/controllers/plugins/simpleSort/simpleSortDescending.controller.intgration.spec.js

```diff
@@ -0,0 +1,181 @@
+          expect(body).to.deep.equal(this.simpleSortModels.map(simpleSortModel => {
+            return {
+              _id: simpleSortModel._id.toString(),
+              name: simpleSortModel.name,
+              createdAt: simpleSortModel.createdAt.toISOString(),
+              updatedAt: simpleSortModel.updatedAt.toISOString(),
+            };
+          }));
+        });
+      });
+
+      describe('user overriding the sort by a different key', () => {
+        beforeEach(() => {
+          this.simpleSortModels = this.simpleSortModels.sort((firstModel, secondModel) => {
+            return secondModel.updatedAt.getTime() - firstModel.updatedAt.getTime();
+          });
+        });
+
+        it('must allow the user to sort updatedAt asc', () => {
+          this.simpleSortModels.reverse();
+          return request(this.app)
+          .get(`${BASE_URI}/?sort=u:asc`)
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.length).to.equal(this.simpleSortModels.length);
+            expect(body).to.deep.equal(this.simpleSortModels.map(simpleSortModel => {
+              return {
+                _id: simpleSortModel._id.toString(),
+                name: simpleSortModel.name,
+                createdAt: simpleSortModel.createdAt.toISOString(),
+                updatedAt: simpleSortModel.updatedAt.toISOString(),
+              };
+            }));
+          });
+        });
+
+        it('must allow the user to sort updatedAt desc', () => {
+          return request(this.app)
+          .get(`${BASE_URI}/?sort=u:desc`)
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.length).to.equal(this.simpleSortModels.length);
+            expect(body).to.deep.equal(this.simpleSortModels.map(simpleSortModel => {
+              return {
+                _id: simpleSortModel._id.toString(),
+                name: simpleSortModel.name,
+                createdAt: simpleSortModel.createdAt.toISOString(),
+                updatedAt: simpleSortModel.updatedAt.toISOString(),
+              };
+            }));
+          });
+        });
+      });
+
+      describe('user overriding the sort by an invalid key', () => {
+        it('must not allow the user to sort name asc', () => {
+          return request(this.app)
+          .get(`${BASE_URI}/?sort=name:asc`)
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.length).to.equal(this.simpleSortModels.length);
+            expect(body).to.deep.equal(this.simpleSortModels.map(simpleSortModel => {
+              return {
+                _id: simpleSortModel._id.toString(),
+                name: simpleSortModel.name,
+                createdAt: simpleSortModel.createdAt.toISOString(),
+                updatedAt: simpleSortModel.updatedAt.toISOString(),
+              };
+            }));
+          });
+        });
+
+        it('must not allow the user to sort name desc', () => {
```

Comment:
Why not? 🤔

Reply:
cuz its not provided here: https://github.com/hubba/curdy/pull/15/files#diff-2f11a382834a9da2c8f4b0676813c8d4R23